README.md

```diff
@@ -91,7 +111,7 @@ TWITTER_OAUTH2_REFRESH_TOKEN
 
 TWITTER_OAUTH2_IS_AUTO_REFRESH_TOKEN - default value is `false`
 
-* OAuth 2.0 Bearer Token (app-only)
+* #### OAuth 2.0 Bearer Token (app-only)
```

Comment:
In the first commit, I put a colon after the little title, I removed and it might work now